plugins/discourse-narrative-bot/lib/discourse_narrative_bot/new_user_narrative.rb

```diff
@@ -182,8 +182,13 @@ def say_hello
       #{instance_eval(&@next_instructions)}
       RAW
 
+      title = I18n.t("#{I18N_KEY}.hello.title", title: SiteSetting.title)
+      if SiteSetting.max_emojis_in_title == 0
+        title = title.gsub(/:\w+:/, '').strip
```

Comment:
- Are emojis in `title` always in their unescaped representation? (as opposed to their Unicode representation)
- What about `:mermaid:t4:`?

Reply:
re the 1st point: primary concern is the default titles as we ship (which all use `:robot:`) - if somebody is modifying the title on their own site they are probably aware of their own site setting restrictions.